src/libraries/System.Runtime.InteropServices/tests/ComDisabled/System.Runtime.InteropServices.Tests.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
```

Comment:
This project should have a unique name  - e.g. `System.Runtime.InteropServices.ComDisabled.Tests.csproj'.

By default, the output path is set based on the project name (see https://github.com/dotnet/arcade/blob/main/src/Microsoft.DotNet.Arcade.Sdk/tools/ProjectLayout.props), so I'm pretty sure this conflicts with the existing `System.Runtime.InteropServices.Tests` and one of them ends up clobbering the other.